app/screens/search/search.js

```diff
@@ -73,11 +74,17 @@ class Search extends PureComponent {
             isFocused: true,
             postId: null,
             preview: false,
-            value: ''
+            value: '',
+            managedConfig: {}
         };
     }
 
+    componentWillMount() {
+        mattermostManaged.addEventListener('change', this.setManagedConfig);
```

Comment:
Event listener needs to be removed on unmount right?

Reply:
done, also removed it from the post_list were you added it before